samples/DataGridListSample.js

```diff
@@ -90,9 +91,11 @@ enyo.kind({
 	mixins: ["moon.SelectionOverlaySupport"],
 	selectionOverlayVerticalOffset: 35,
 	subCaption: "Sub Caption",
+	imageSizing: "cover",
 	bindings: [
 		{from: ".model.text", to: ".caption"},
 		{from: ".model.subText", to: ".subCaption"},
-		{from: ".model.url", to: ".source"}
+		{from: ".model.url", to: ".source"},
+		{from: ".model.placeholder", to: ".placeholder"}
```

Comment:
Please remove the prefixed `.` in front of each of these bindings to/from strings. It is no longer needed.